e2e/test/iothub/twin/TwinE2ETests.cs

```diff
@@ -19,6 +20,7 @@ namespace Microsoft.Azure.Devices.E2ETests.Twins
     public class TwinE2ETests : E2EMsTestBase
     {
         private readonly string _devicePrefix = $"E2E_{nameof(TwinE2ETests)}_";
+        private readonly List<object> _listOfPropertyValues = new List<object> { 1, "someString", false, new CustomTwinProperty { Id = 123, Name = "someName" } };
```

Comment:
Consider formatting this for easier reading:

```csharp
private readonly List<object> _listOfPropertyValues = List<object>
{
    1,
    "someString",
    //...
};
```

Reply:
sure